.circleci/config.yml

```diff
@@ -71,15 +62,15 @@ workflows:
               only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/
           matrix:
             parameters:
-              ruby-version: ["2.7.1", "2.6", "2.5"]
-              vault-version: ["1.5.0", "1.4.2", "1.4.1", "1.4.0", "1.3.6"]
+              ruby-version: ["3.1.2", "3.0.4", "2.7.6", "2.6.10"]
+              vault-version: ["1.11.1", "1.10.5", "1.9.8"]
           name: test-ruby-<< matrix.ruby-version >>-vault-<< matrix.vault-version >>
       - build-release:
           requires:
             - test
           context: vault-gem-release
           filters:
             tags:
-              only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/ 
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/
```

Comment:
I noticed this removed trailing space - just want to confirm this was intentional and not the IDE removing something that's potentially desired?